marklogic-data-hub/src/test/java/com/marklogic/hub/flow/TemporalWriteTest.java

```diff
@@ -17,11 +19,18 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.util.stream.Stream;
+
 import static com.marklogic.client.io.DocumentMetadataHandle.Capability.*;
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class TemporalWriteTest extends AbstractHubCoreTest {
 
+    private InputEndpoint.BulkInputCaller bulkInputCaller;
```

Comment:
We don't want this as a field, as it's only used by one test as a local variable.

Reply:
Using it as localvariable